.infrastructure/09_iam_role_deploy.tf

```diff
@@ -0,0 +1,149 @@
+data "aws_caller_identity" "current" {}
+
+###############################################################################
+#                      IAM Role to use on deploy website                       #
+###############################################################################
+resource "aws_iam_role" "deploy_website" {
+  name        = "GitHubActionDeployWebsite"
+  description = "Role to assume to deploy the website"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Effect = "Allow",
+        Principal = {
+          "Federated" : "arn:aws:iam::${data.aws_caller_identity.current.account_id}:oidc-provider/token.actions.githubusercontent.com"
+        },
+        Action = "sts:AssumeRoleWithWebIdentity",
+        Condition = {
+          StringLike = {
+            "token.actions.githubusercontent.com:sub" : "repo:${var.github_repository}:*"
+          },
+          "ForAllValues:StringEquals" = {
+            "token.actions.githubusercontent.com:iss" : "https://token.actions.githubusercontent.com",
+            "token.actions.githubusercontent.com:aud" : "sts.amazonaws.com"
+          }
+        }
+      }
+    ]
+  })
+}
+
+resource "aws_iam_policy" "deploy_website" {
+  name        = "DeployWebsite"
+  description = "Policy to allow to deploy the website"
+
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = [
+          "s3:PutObject",
+          "s3:GetObject",
+          "s3:PutObjectAcl",
+          "s3:DeleteObject"
+        ]
+        Effect = "Allow"
+        Resource = [
+          format("%s/*", aws_s3_bucket.website.arn)
+        ]
+      },
+      {
+        Action = [
+          "s3:ListBucket"
+        ]
+        Effect = "Allow"
+        Resource = [
+          aws_s3_bucket.website.arn
+        ]
+      },
+      {
+        Action = [
+          "cloudfront:CreateInvalidation"
+        ]
+        Effect = "Allow"
+        Resource = [
+          aws_cloudfront_distribution.website.arn
+        ]
+      }
+    ]
+  })
+}
+
+resource "aws_iam_role_policy_attachment" "deploy_website" {
+  role       = aws_iam_role.deploy_website.name
+  policy_arn = aws_iam_policy.deploy_website.arn
+}
+
+###############################################################################
+#                      IAM Role to use on deploy CMS                       #
+###############################################################################
+resource "aws_iam_role" "deploy_ecs" {
+  name        = "GitHubActionDeployECS"
+  description = "Role to assume to deploy on ECS."
+
+
+  assume_role_policy = jsonencode({
```

Comment:
This policy is the same as [this one](https://github.com/pagopa/b2b-portals/pull/58/files#diff-f9af473ca5e59e9c0f9867018d548d1eb8c2fa0183ddc85d83fa222870a493e8R10-R30), can we remove the duplication?

Reply:
The policy is defined with JSON within each role. I create it externally as a policy and link it to both roles, okay?

Reply:
change present in the latest commit